qc/collect_hs_metrics.cwl

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "collect HS metrics"
+baseCommand: ["/usr/bin/java", "-Xmx16g", "-jar", "/usr/picard/picard.jar", "CollectHsMetrics"]
+arguments:
+    ["O=", { valueFrom: $(runtime.outdir)/HsMetrics.txt }]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "registry.gsc.wustl.edu/genome/picard-2.4.1-r:2"
+inputs:
+    bam:
+        type: File
+        inputBinding:
+            prefix: "I="
+        secondaryFiles: [.bai]
+    reference:
+        type: File
+        inputBinding:
+            prefix: "R="
+        secondaryFiles: [.fai]
+    bait_intervals:
+        type: File
+        inputBinding:
+            prefix: "BAIT_INTERVALS="
+    target_intervals:
+        type: File
+        inputBinding:
+            prefix: "TARGET_INTERVALS="
+outputs:
```

Comment:
Sorry @susannasiebert, I missed the comment above about not implementing optional features yet(which is fine). I was aware CWL had the ability to have optional outputs but I wasn't sure how they are connected to an input param to turn that feature on/off. Does the name of the file need to be provided as an input or a Boolean arg?